### PR TITLE
Added the latest version(3.0.3) link

### PR DESCRIPTION
**Pull request by sapinderpalsingh**
Added latest uber jar link.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -140,6 +140,7 @@ You can build and/or use the maven coordinates to work with `azure-cosmosdb-spar
 
 | Spark | Scala | Latest version |
 |---|---|---|
+| 2.4.0 | 2.11 | [azure-cosmosdb-spark_2.4.0_2.11_3.0.3](https://search.maven.org/remotecontent?filepath=com/microsoft/azure/azure-cosmosdb-spark_2.4.0_2.11/3.0.3/azure-cosmosdb-spark_2.4.0_2.11-3.0.3-uber.jar)
 | 2.4.0 | 2.11 | [azure-cosmosdb-spark_2.4.0_2.11_2.1.2](https://repo1.maven.org/maven2/com/microsoft/azure/azure-cosmosdb-spark_2.4.0_2.11/2.1.2/azure-cosmosdb-spark_2.4.0_2.11-2.1.2-uber.jar)
```
Comment (FabianMeiswinkel) [inline README.md]: Please remove this link (only the latest for each Spark version is supposed to be listed here)

File: README.md [hunk 2]
```diff
@@ -140,6 +140,7 @@ You can build and/or use the maven coordinates to work with `azure-cosmosdb-spar
 
 | Spark | Scala | Latest version |
 |---|---|---|
+| 2.4.0 | 2.11 | [azure-cosmosdb-spark_2.4.0_2.11_3.0.3](https://search.maven.org/remotecontent?filepath=com/microsoft/azure/azure-cosmosdb-spark_2.4.0_2.11/3.0.3/azure-cosmosdb-spark_2.4.0_2.11-3.0.3-uber.jar)
```
Comment (FabianMeiswinkel) [inline README.md]: Please use this link instead - we will make sure that it points tot he "last known good" going forward.

https://aka.ms/CosmosDB_OLTP_Spark_2.4_LKG

**Review by FabianMeiswinkel — CHANGES_REQUESTED** [review]
Thanks for taking care of this!

File: README.md (hunk 2 above)
Comment (sapinderpalsingh) [reply README.md]: Thanks Fabian, so should I close the PR or or just remove all the SDK links and just point to the above link that you shared?

File: README.md (hunk 1 above)
Comment (sapinderpalsingh) [reply README.md]: Ok, i will update it.

File: README.md (hunk 2 above)
Comment (FabianMeiswinkel) [reply README.md]: I would just leave the links for Spark 2.1 / 2.2 and 2.3 untouched and replace the link for Spark 2.4 with the one above. Thanks!

**Review by FabianMeiswinkel — APPROVED** [review]

**Pull request merged**